src/System.Collections/tests/Performance/System.Collections.Performance.Tests.csproj

```diff
@@ -9,6 +9,7 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' " />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' " />
   <ItemGroup>
+    <Compile Include="$(CommonTestPath)\System\Collections\DictionaryExtensions.cs" />
```

Comment:
as Dan mentioned please add the Link element for having VS show such files nicely in the project view.